config/MasterTranslations.xml

```diff
@@ -625,11 +625,11 @@
 			<Text language="de"><![CDATA[Versatz für den Ballensammler]]></Text>
 			<Text language="en"><![CDATA[Offset for the bale collector]]></Text>
 		</Translation>
-		<Translation name="CP_vehicle_setting_toolOffsetZ_title">
+		<Translation name="CP_vehicle_setting_combineOffsetZ_title">
 			<Text language="de"><![CDATA[Geräteversatz vertikal]]></Text>
 			<Text language="en"><![CDATA[Tool offset vertical]]></Text>
```

Comment:
Unloader offset forward/backward?

Reply:
Would just write "Offset forward/backward" to save some space.
Same for the other, "Offset left/right".